### PR TITLE
docs: update the description of the coveredCall contract.

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
update the description of the coveredCall contract and the link to the source.

### CODE REVIEW

File: main/zoe/guide/contracts/covered-call.md [hunk 1]
```diff
@@ -2,160 +2,227 @@
...
-  bobProposalSwap,
-  bobPayments,
-);
-
-const daveSwapInvite = await bobSwapInviteP;
+const daveSwapInvitation = bobSwapSeat.getOfferResult();
 ```
 
-Bob now has a call option, in the form of an invite, ready to sell via an atomic swap contract.
+Bob now has a call option that he can sell via an atomic swap contract.
```
Comment (katelynsills) [inline main/zoe/guide/contracts/covered-call.md:164]: Hmm, this was in the original but doesn't seem right. Should we move it up before the atomic swap?

File: main/zoe/guide/contracts/covered-call.md [hunk 2]
```diff
@@ -2,160 +2,227 @@
...
 const {
-  value: [{ instanceHandle: swapInstanceHandle }],
-} = inviteIssuer.getAmountOf(daveSwapInvite);
-
-const {
-  installationHandle: daveSwapInstallId,
-  issuerKeywordRecord: daveSwapIssuers,
-} = zoe.getInstance(swapInstanceHandle);
+  installation: daveSwapInstall,
+  instance,
```
Comment (katelynsills) [inline main/zoe/guide/contracts/covered-call.md:178]: do we need the instance?

**Review by katelynsills — APPROVED** [review]
A few changes, but otherwise looks good!

File: main/zoe/guide/contracts/covered-call.md [hunk 3]
```diff
@@ -2,160 +2,227 @@
...
 
-##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/958a2c0a3dec38bdba2234934119ea2c28958262/packages/zoe/src/contracts/coveredCall.js) (Last updated: 4/22/2020)
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/f29591519809dbadf19db0a26f38704d87429b89/packages/zoe/src/contracts/coveredCall.js) (Last updated: 9/12/2020)
 ##### [View all contracts on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts)
 
-::: tip Out-of-date status
-Zoe's master branch is currently an Alpha release candidate. This doc
-and its underlying contract are in the process of being updated, and should be current with the release candidate in another few days. What you see here is out of date. We apologize for any inconvenience this may cause.
-:::
+In a covered call, the owner of a digital asset sells a call option. A call
```
[the remark this answers is not in the document]
Comment (Chris-Hibbert) [reply main/zoe/guide/contracts/covered-call.md]: good point.

File: main/zoe/guide/contracts/covered-call.md [hunk 4]
```diff
@@ -2,160 +2,227 @@
...
+contract is cancelled. It is "covered", meaning that the assets it
+describes is in the seller's possession, and the actual asset will be
+transfered when the exchange takes place.
+
+In this contract, the expiry date is represented by the deadline at which
+the owner of the digital asset's offer is cancelled. Therefore, the owner
+of the digital asset's proposal must have an `exit` of "afterDeadline".
+
+The offer result from the creation of the covered call is another
+invitation that serves as the call option. The invitation includes details
```
[the remark this answers is not in the document]
Comment (Chris-Hibbert) [reply main/zoe/guide/contracts/covered-call.md]: yes. Fixed, I think.

File: main/zoe/guide/contracts/covered-call.md (hunk 2 above)
Comment (Chris-Hibbert) [reply main/zoe/guide/contracts/covered-call.md:180]: Dave should validate the install and the issuers. I've added that.

File: main/zoe/guide/contracts/covered-call.md (hunk 1 above)
Comment (Chris-Hibbert) [reply main/zoe/guide/contracts/covered-call.md]: Once I moved it closer to where it applied, it looked redundant, so I dropped it.

File: main/zoe/guide/contracts/covered-call.md [hunk 5]
```diff
@@ -2,160 +2,227 @@
...
+refund of the underlying assets (as mentioned above) or payments in the
+amount of the strike price. Zoe's enforcement of offer safety guarantees
+that the payout is either a refund or payments in the amount of the strike
+price, regardless of whether the contract is buggy.
 
-In a covered call, the owner of a digital asset sells a call option. A call option is the right to buy the digital asset at a certain price, called the strike price. The call option has an expiry date, at which point the contract is cancelled.
+The offerResult of the initial seat resolves to the call option itself: an
+inspectable invitation to buy the underlying assets. The call option
+invitation has this additional information in the value: `{ expirationDate,
+timeAuthority, underlyingAssets, strikePrice }`.
```
[the remark this answers is not in the document]
Comment (Chris-Hibbert) [reply main/zoe/guide/contracts/covered-call.md:72]: cleaned up the earlier section before I saw this comment. They now say similar things differently, so I think it's the helpful kind of redundancy.

**Pull request merged**